### DISCUSSION TITLE
User/ Customer Login

### DISCUSSION THREAD

**Discussion by hrish09** [lunarphp/lunar#1689] [Q&A] [answered]
Is there any plugin or package which provides the user login where one can see his/her previous orders and all his/her past history will be tracked

**Comment by alecritson** [comment, marked as answer]
You thinking you want customers to be able to log in to the panel? If so this wasn't the intention of the panel and so would need to be implemented yourself.

Otherwise it's quite storefront specific so if you have an account area on your website you can quite easily implement this yourself. We're looking at developing a storefront developers can install which will come with all these goodies in the future :)

**Reply by hrish09** [reply]
Thank you @alecritson can you update me with any package like jetstream / breeze which will support lunar

**Reply by alecritson** [reply]
You can use whatever authentication package you want for your storefront, we've used Fortify for our own sites without any issues :)
